odk1-src/form-question-types.rst

```diff
@@ -1397,16 +1397,21 @@ An image widget that does not include a :guilabel:`Choose Image` button. This re
   
 .. _selfie-widget:
 
-Selfie widget
-~~~~~~~~~~~~~~~
+Self portrait (*selfie*) widget
```

Comment:
I think if we aren't calling it selfie, then we should remove the references to selfie in this document and also in the all-widgets form. What do you think, @lognaturel?

Reply:
I think it is possible people will still look for it under this name, so I think including the word "selfie" is helpful.